### ISSUE TITLE
Option to disable Confirmation dialog

### ISSUE THREAD

**Issue by jazzybeat** [simov/screenshot-capture#12]
This tool is simple and fast. There is just one annoying dialog that appears if you capture screens directly to the [clipboard](https://drive.google.com/file/d/1LbtM_P81eXiqhlitxtsk0EV1KV1AF3ob/view?usp=sharing]). 

Screenshot Capture:
Binary Image
Saved to Clipboard!

It would be great if you could include the option don't show this again or just disable in settings.

**Comment by simov** [comment]
That popup message was the easiest way to provide any visual feedback to the user that the operation succeeded, though I can see how that can be annoying as well since you have to click on it or hit esc key.

**Comment by vladimirgamalyan** [comment]
Well, in most cases, it's reasonable to show a confirmation popup. However, sometimes when you need to take a lot of shots to the clipboard (and then paste, for example, to PS), it's a little annoying extra step (in any case, the user will see the result in the paste step).

**Comment by ScottESanDiego** [comment]
Agreed - the confirmation dialog is good for debugging, but seems to get in the way during daily use.  An option to disable would make the extension more usable.

**Comment by simov** [comment]
The option to disable the confirmation dialog when copying into the clipboard landed in v3.1, check out the extension's options page. You should receive the update shortly or if you have it installed manually you can download the latest release from here https://github.com/simov/screenshot-capture/releases
